libp2p/crypto/rsa.py

```diff
@@ -30,6 +35,11 @@ def new(cls, bits: int = 2048, e: int = 65537) -> "RSAPrivateKey":
     def to_bytes(self) -> bytes:
         return self.impl.export_key("DER")
 
+    @classmethod
+    def from_bytes(cls, key_bytes: bytes) -> "RSAPrivateKey":
```

Comment:
Probably yes if we want to initialize a `RSAPrivateKey` with raw bytes? I added this just from the symmetricity. I can remove it if you think it's better.

Reply:
Yeah I guess. I saw that it's added in `RSAPrivateKey` but not `Secp256k1PrivateKey` so wonder why.

Reply:
It is because `Secp256k1PrivateKey` already has `new`, which functions exactly as the `from_bytes`: https://github.com/libp2p/py-libp2p/pull/253/files/6177024d4fb9eaf9fc6244cf1acacc1ae2742ad6#diff-35f98b0fb8106eef634a5c8ae53c9494R30